ltr/concourse/fetch.sh

```diff
@@ -59,8 +59,8 @@ ssh -i /tmp/concourse_ssh_key -o StrictHostKeyChecking=no "ubuntu@${instance_ip}
 
   mkdir tmp
   set +x
-  echo "running bundle exec rake learn_to_rank:fetch_bigquery_export[BIGQUERY_CREDENTIALS,bigquery]"
-  \$docker bundle exec rake 'learn_to_rank:fetch_bigquery_export[$BIGQUERY_CREDENTIALS,bigquery]'
+  echo "running bundle exec rake learn_to_rank:fetch_bigquery_export[bigquery]"
```

Comment:
Is it safe to remove the `set +x`/`set -x` lines now?

Reply:
Yes it should be, I'll do it in a followup PR 🙂